src/components/non-bitgo.js

```diff
@@ -118,7 +118,7 @@ class NonBitGoRecoveryForm extends Component {
         return obj;
       }, {});
 
-      if (this.state.coin === 'bsv' && this.state.apiKey) {
+      if ((this.state.coin === 'bsv' || this.state.coin === 'bch') && this.state.apiKey) {
```

Comment:
If you have lodash, you can filter the `requiredParams` array by those items containing `apiKey` and then get the key, so it would be something like: `if (coinsWithApiKey().contain(this.state.coin) && this.state.apiKey) { ... }` where `coinsWithApiKey()` returns the keys from `requiredParams`that contain `apiKey`.

Reply:
or you could even do `coinsWithParam('apiKey')`

Reply:
https://lodash.com/docs/4.17.15#pickBy